ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -107,43 +107,11 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         case primary
         case system
         case custom
-
-        func tintColor(fluentTheme: FluentTheme) -> UIColor {
-            switch self {
-            case .primary, .default, .custom:
-                return UIColor(light: fluentTheme.color(.foregroundOnColor).light,
-                               dark: fluentTheme.color(.foreground2).dark)
-            case .system:
-                return fluentTheme.color(.foreground2)
-            }
-        }
-
-        func titleColor(fluentTheme: FluentTheme) -> UIColor {
-            switch self {
-            case .primary, .default, .custom:
-                return UIColor(light: fluentTheme.color(.foregroundOnColor).light,
-                               dark: fluentTheme.color(.foreground1).dark)
-            case .system:
-                return fluentTheme.color(.foreground1)
-            }
-        }
-
-        public func backgroundColor(fluentTheme: FluentTheme, customColor: UIColor? = nil) -> UIColor {
-            let defaultColor = UIColor(light: fluentTheme.color(.brandBackground1).light,
-                                       dark: fluentTheme.color(.background3).dark)
-            switch self {
-            case .primary, .default:
-                return defaultColor
-            case .system:
-                return fluentTheme.color(.background3)
-            case .custom:
-                return customColor ?? defaultColor
-            }
-        }
     }
 
+    @available(*, deprecated, renamed: "backgroundColor(forStyle:theme:)", message: "NavigationBar has been tokenized. Use the replacement method, passing in .system for the style.")
```

Comment:
Unfortunately, since control tokens cannot be used in Objective C, and we don't have a plan for when they will be available, I don't think we can deprecate this API yet.

Reply:
Why can't we deprecate this? Even if we can't use control tokens in Objective-C, `NavigationBar.Style` and `FluentTheme` should be fair game.